app/features/atp/identity-verification/ui/idx/controller.ts

```diff
@@ -3,7 +3,7 @@ import { PageSetup } from "../../../../../interfaces/PageSetup";
 import { pathName } from "../../../../../paths";
 import Logger from "../../../../../utils/logger";
 
-const template = "atp/identity-evidence/ui/idx/view.njk";
+const template = "atp/identity-verification/ui/idx/view.njk";
```

Comment:
Should this be `atp/identity-history` here?

Reply:
No this is a mistake that I spotted, it's for the identity-verification ui.